### DISCUSSION TITLE
KPC is adding my personal videos library into my Movies library

### DISCUSSION THREAD

**Discussion by KelSceptic** [croneter/PlexKodiConnect#1579] [Q&A]
Hello,

In Plex I have 2 separate libraries with 2 separate paths, one is Movies /media/Movies and the other is Home Videos /media/Home Videos.
The Home Videos library is using the Scanner: Plex Video Files with the Agent: Personal Media.

While in Plex I have two distinct libraries, in Kodi, with KPC, my personal videos are showing up in my Movies library.

Any ideea on how to fix this, please?

Thank you.

**Comment by croneter** [comment]
Yes, see here: https://github.com/croneter/PlexKodiConnect/wiki/Video-Nodes-and-Skins
